O365DataRetriever.ps1

```diff
@@ -700,7 +700,21 @@ $ConnectButton.Add_Click( {
 			Write-Host "Microsoft Teams Module present. OK." -ForegroundColor Green
 		}
 
-        
+		#Check if Microsoft Exchange Online module present
```

Comment:
@swngdnz : Trying with a non-MFA user completely stops the process. And I also have the EXO MFA module already installed on my machine.

![error_non-mfa](https://user-images.githubusercontent.com/25181757/44915809-e14e3c80-ad2b-11e8-9119-1491783e9b2b.png)